### DISCUSSION TITLE
validator return null if not a data

### DISCUSSION THREAD

**Discussion by sergefabre** [adonisjs/core#3698] [Help]
Hello
I need your help

How it's possible than tha validator return Null if one data missing ?

**Comment by feniljariwala82** [comment]
Can you please show us the code?

**Reply by sergefabre** [reply]
`export default class DptProdValidator {
  constructor(protected ctx: HttpContextContract) {}

  public schema = schema.create(DptProdValidator.schemaObject)
  public static schemaObject = {
    code: schema.string.nullableAndOptional(),
    nom: schema.string(),
    periode_validite: schema.object().members({
      date_debut_validite: schema.date.optional({ format: 'yyyy-MM-dd' }),
      date_fin_validite: schema.date.optional({ format: 'yyyy-MM-dd' }),
      date_creation: schema.date.optional({ format: 'iso' }),
      date_modification: schema.date.optional({ format: 'iso' }),
    }),
  }
  public messages = {}
}`
